### PR TITLE
Adaptive toolbar

### PR DESCRIPTION
**Pull request by magdalenaan**
part of: https://github.com/telerik/kendo-themes-private/issues/96
closes: https://github.com/telerik/kendo-themes-private/issues/160

### CODE REVIEW

File: packages/bootstrap/scss/action-sheet/_variables.scss [hunk 1]
```diff
@@ -72,6 +72,7 @@ $kendo-actionsheet-item-disabled-shadow: null !default;
 
 
 // Adaptive Actionsheet
+$kendo-adaptive-actionsheet-font-size: $kendo-font-size-lg !default;
```
Comment (Juveniel) [inline packages/bootstrap/scss/action-sheet/_variables.scss:75]: Is this change intended by design ?

**Comment by Juveniel** [comment]
There are also some lint errors - see the `Lint Scripts` CI.

File: packages/bootstrap/scss/action-sheet/_variables.scss (hunk 1 above)
Comment (magdalenaan) [reply packages/bootstrap/scss/action-sheet/_variables.scss:75]: I have discussed this topic with @dmanova: the default font size in adaptive mode is 16px.

**Review by Juveniel — APPROVED** [review]
